src/GitVersionTask/AssemblyInfoBuilder/AssemblyInfoBuilder.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+
 using GitVersion;
 
 public class AssemblyInfoBuilder
 {
     public string GetAssemblyInfoText(VersionVariables vars)
     {
+        var v = vars.ToArray();
```

Comment:
Ah yes. I just like arrays better due to the built in syntactic sugar. Not that I used that here, but habits die hard. :)